src/main/java/com/google/sps/servlets/GrantLocationServlet.java

```diff
@@ -5,6 +5,14 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
 
 @WebServlet("/grant-location")
 public class GrantLocationServlet extends HttpServlet {
```

Comment:
Validate that we get this data at all.  I'm curious if granting location access is something that is between google maps and the user or not.